kolibri/core/assets/src/api-resource.js

```diff
@@ -215,7 +215,7 @@ class Model {
 
   set(attributes) {
     // force IDs to always be strings - this should be changed on the server-side too
-    if (this.resource.idKey in attributes) {
+    if (attributes && this.resource.idKey in attributes) {
```

Comment:
Ah, this might explain the issue that @ralphiee22 was having also.

@ralphiee22 probably good to revert the change we made, and pull in this from master when it is merged.